partner_academic_title/views/partner_academic_title_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_academic_title_form" model="ir.ui.view">
+            <field name="name">partner.academic.title.form</field>
+            <field name="model">partner.academic.title</field>
+            <field name="arch" type="xml">
+                <form>
+                    <sheet>
```

Comment:
Maybe a nitpick, but IMO `<sheet>` is to be used in "business documents" and not in code tables.